src/components/grids/TabbedDisplay/index.tsx

```diff
@@ -36,12 +36,12 @@ const DEFAULT_STYLE: TabbedDisplayStyleSpec = {
   },
   inactive: {
     backgroundColor: 'transparent',
-    textColor: gray[400],
+    textColor: gray[900],
```

Comment:
What do you think about using `gray[900]` for the active tab's text color as well? I personally think it looks better given these changes.

Reply:
@jernestmyers I don't have any objection so will change it 👍 Though, it may not be that noticeably different because of the presence of indicatorColor.

Reply:
@jernestmyers just changed the active's text color to be `gray[900]`